### ISSUE TITLE
java.lang.NoSuchFieldException: a

### ISSUE THREAD

**Issue by mibby** [elBukkit/MagicPlugin#431]
Magic dev 2797
Paper dev 387 (Spigot 1.13.2)

```
[00:52:46] [Server thread/INFO]: [Magic] Enabling Magic v7.6.15-SNAPSHOT-1fbe3b1
[00:52:46] [Server thread/ERROR]: An unexpected error occurred initializing Magic
java.lang.NoSuchFieldException: a
	at java.lang.Class.getField(Class.java:1703) ~[?:1.8.0_192]
	at com.elmakers.mine.bukkit.utility.NMSUtils.initialize(NMSUtils.java:474) ~[?:?]
	at com.elmakers.mine.bukkit.magic.MagicPlugin.onEnable(MagicPlugin.java:171) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[patched_1.13.2.jar:git-Paper-387]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:325) ~[patched_1.13.2.jar:git-Paper-387]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) ~[patched_1.13.2.jar:git-Paper-387]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:438) ~[patched_1.13.2.jar:git-Paper-387]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:352) ~[patched_1.13.2.jar:git-Paper-387]
	at net.minecraft.server.v1_13_R2.MinecraftServer.l(MinecraftServer.java:613) ~[patched_1.13.2.jar:git-Paper-387]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:575) ~[patched_1.13.2.jar:git-Paper-387]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:435) ~[patched_1.13.2.jar:git-Paper-387]
	at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:316) ~[patched_1.13.2.jar:git-Paper-387]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:791) ~[patched_1.13.2.jar:git-Paper-387]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_192]
[00:52:46] [Server thread/INFO]: [Magic] Something went wrong with some Deep Magic, plugin will not load.
[00:52:46] [Server thread/INFO]: [Magic] Please make sure you are running a compatible version of Spigot (1.9 or Higher)!
```

**Comment by mibby** [comment]
https://github.com/elBukkit/MagicPlugin/blob/56eca4750a28e606e3580379be198c34a1cae048/CompatibilityLib/src/main/java/com/elmakers/mine/bukkit/utility/NMSUtils.java#L474 needs to be updated to minX, minY, minZ, maxX, maxY, & maxZ for a-f.

**Comment by NathanWolf** [comment]
Thanks for the head's up! I just released 7.6.15 with 1.13.2 support.
